pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -355,7 +355,7 @@ export default defineComponent({
       disabled: isDisabled,
       [status]: status,
       suffix: hasSuffix,
-      'v-popper--has-tooltip': hasTooltip,
+      'has-clean-tooltip': hasTooltip,
```

Comment:
there;s a couple of references to v-popper--has-tooltip still in cypress/e2e/tests/pages/explorer2/namespace-picker.spec.ts, do those need updating?

(starting review, got pulled off to something else, didn't want to lose this comment)